services/src/main/java/org/keycloak/services/resources/admin/UserResource.java

```diff
@@ -753,32 +753,28 @@ public Response resetPasswordEmail(@QueryParam(OIDCLoginProtocol.REDIRECT_URI_PA
         return executeActionsEmail(redirectUri, clientId, null, actions);
     }
 
-
     /**
-     * Send a update account email to the user
-     *
-     * An email contains a link the user can click to perform a set of required actions.
-     * The redirectUri and clientId parameters are optional. If no redirect is given, then there will
-     * be no link back to click after actions have completed.  Redirect uri must be a valid uri for the
-     * particular clientId.
-     *
+     * Create action token 
+     * 
+     * 
+     * An token to perform a set of required actions. The clientId parameters are
```

Comment:
Suggested rewording : "**A** token ... . The clientId parameter **is** optional"